src/test/java/seedu/address/logic/parser/CovigentAppParserTest.java

```diff
@@ -65,6 +67,19 @@ public void parseCommand_editPatient() throws Exception {
         assertEquals(new EditPatientCommand(patient.getName().toString(), descriptor), command);
     }
 
+    @Test
+    public void parseCommand_editTask() throws Exception {
```

Comment:
would it be better to name the test `parseCommand_editTask_failure()` or `parseCommand_editTask_throwExceptions()` instead? Same for the tests at the bottom too!

Reply:
Good point. We should change the names of the tests here!